src/components/popup-dialog/PopupDialog.tsx

```diff
@@ -45,6 +45,7 @@ const PopupDialog: FC<PopupDialogProps> = ({
       data-testid='popup'
       fullScreen={isMobile}
       maxWidth='xl'
+      onClick={(e) => e.stopPropagation()}
```

Comment:
I guess, we should add leave confirmation when data is filled in the modal, not disable default behaviour